source/Calamari/Kubernetes/Scripts/KubectlBashContext.sh

```diff
@@ -110,83 +131,97 @@ function setup_context {
       K8S_Azure_Cluster+="-admin"
     fi
     kubectl config set-context $K8S_Azure_Cluster --namespace=$Octopus_K8S_Namespace
+  elif [[ $IsUsingPodServiceAccount ]]; then
```

Comment:
caught this during testing. This check will return `true` as long as `IsUsingPodServiceAccount ` has a value, regardless of `true` or `false`

Reply:
Nice catch